code/modules/crafting/craft.dm

```diff
@@ -1,24 +1,35 @@
 /datum/personal_crafting
 	var/busy
 	var/viewing_category = 1 //typical powergamer starting on the Weapons tab
-	var/list/categories = list(CAT_WEAPON,
-				CAT_AMMO,
+	var/viewing_subcategory = 1
+	var/list/categories = list(
+				CAT_WEAPONRY,
 				CAT_ROBOT,
 				CAT_MISC,
 				CAT_PRIMAL,
-				CAT_BREAD,
-				CAT_BURGER,
-				CAT_CAKE,
-				CAT_EGG,
-				CAT_MEAT,
-				CAT_MISCFOOD,
-				CAT_PASTRY,
-				CAT_PIE,
-				CAT_PIZZA,
-				CAT_SALAD,
-				CAT_SANDWICH,
-				CAT_SOUP,
-				CAT_SPAGHETTI)
+				CAT_FOOD)
+	var/list/subcategories = list(
```

Comment:
Actually there's nothing stopping you from doing something like:

var/list/categories = list(
1 = list(5, 6, 7),
2 = 0,
3 = 0,
4 = list(8, 9, 10, 11, 12, 13, 14, 15)
)